examples/java/README.md

```diff
@@ -1,13 +1,10 @@
 # Java Example App
 
-Eclipse project for developing a MEGA app for Java.
+Eclipse project for developing a MEGA app in Java.
 
 This example shows a simple GUI (Swing) to allow the user log into MEGA. If login is successfull, it retrieves the MEGA filesystem and lists files/folders in the root folder.
 
```

Comment:
It might be good to list required build dependencies here. Off the top of my head I can think of the following (good, it's a short list):
* compiler, build tool chain (on Linux probably mostly in `build-essential`)
* SWIG (what version)
* J2SDK (what version? I'd guess >= Java 7)
* anything else?